gen_cert.py

```diff
@@ -101,8 +101,18 @@ def get_cert_date(force_date, locale, timezone):
 class CertificateGen(object):
     """Manages the pdf, signatures, and S3 bucket for course certificates."""
 
-    def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
-                 dir_prefix=None, long_org=None, long_course=None, issued_date=None):
+    def __init__(
+        self,
+        course_id,
+        aws_id=None,
+        aws_key=None,
+        designation=None,
+        dir_prefix=None,
+        long_course=None,
+        long_org=None,
+        issued_date=None,
+        template_pdf=None,
```

Comment:
No need to jump to conclusions.  I checked this when I made the change and have no issue reverting it.  I alphabetized out of habit.